### PR TITLE
doubles the prize money for slots

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
tweaked how much pay you get from slots
now 
its 
5->10
400->800
1000->2000
10000->20000
#### Changelog

:cl:  
tweak: tweaked slots payment
/:cl:

### CODE REVIEW

**Comment by ynot01** [comment]
make slots viable for money generation maybe people will actually use them

**Comment by boodaliboo** [comment]
> make slots viable for money generation maybe people will actually use them

what do you mean by this i use them

File: code/game/machinery/slotmachine.dm [hunk 1]
```diff
@@ -4,10 +4,10 @@
...
 \*******************************/
 
-#define SPIN_PRICE 5
-#define SMALL_PRIZE 400
-#define BIG_PRIZE 1000
-#define JACKPOT 10000
+#define SPIN_PRICE 10
+#define SMALL_PRIZE 800
+#define BIG_PRIZE 2000
+#define JACKPOT 20000
```
Comment (ThatLing) [inline code/game/machinery/slotmachine.dm:10]: You forgot to change the flavourtext that gets displayed when you win

**Review by ThatLing — CHANGES_REQUESTED** [review]

**Comment by ThatLing** [comment]
There is really no need for this, you can already easily steamroll into becoming a millionaire if you are lucky. This just increases the rate in which you earn money, not the chance of striking it rich.

Everyone that upvoted the PR just wants an easy way of getting a lot of money. Why don't we just make people spawn with a million spacebucks instead?

**Comment by alexkar598** [comment]
>Make slots viable money generation

..... I have a feeling some people dont know the meaning of gambling...

**Pull request closed without merging**